cime/config/e3sm/config_grids.xml

```diff
@@ -2145,6 +2184,36 @@
       <map name="OCN2ATM_SMAPNAME">cpl/cpl6/map_oRRS18to6_TO_enax4v1_aave.170620.nc</map>
     </gridmap>
 
+    <gridmap atm_grid="ne0np4_northamericax4v1" ocn_grid="oRRS15to5">
+      <map name="ATM2OCN_FMAPNAME">cpl/gridmaps/northamericax4v1np4/map_northamericax4v1_to_oRRS15to5_mono.190613.nc</map>
+      <map name="ATM2OCN_SMAPNAME">cpl/gridmaps/northamericax4v1np4/map_northamericax4v1_to_oRRS15to5_mono.190613.nc</map>
```

Comment:
I think the SMAP (mapping weights for state variables) should use the highorder maps, as opposed to the mono maps. Probably the same for the VMAPs.